willeswa/sendit-app
# #161583001 view order details

What does this PR do?
Have table and navigation mockup for the show page
Description of Task to be completed?
Created a show page to show the details of an existing order. 
Users have the ability to cancel the order or updated information 
How should this be manually tested?
Clone this repo into your local computer.
Navigate to the UI folder and open the show.html file. If you open the file in a web browser, you can see the links that would be implemented at a later stage to allow for the required functionality
Any background context you want to provide?
Some functionality will be implemented during the implementation of the backend
What are the relevant pivotal tracker stories?
#161583001
Screenshots (if appropriate)
![screenshot from 2018-11-02 16-53-09](https://user-images.githubusercontent.com/38868860/47919251-fa0bd780-debf-11e8-91e7-e88e55082396.png)
